tests/utils/sandbox-child.mjs

```diff
@@ -13,16 +18,29 @@ process.on(
   async (msg) => {
     if (msg?.action === 'exit') {
       process.exit(0)
+    } else if (msg?.action === 'loadFunction') {
+      const [ctx, options] = msg.args
+
+      invokeFunctionImpl = await loadFunction(ctx, options)
+
+      if (process.send) {
```

Comment:
when is this false? is this expected?

Reply:
typescript `@types/node` report it as nullable in general which forces some checking (due to added `// @ts-check` directive)

generally it won't exist in parent process or if there is no IPC bridge setup (this is to send IPC messages to parent process), but in our current usage it will actually always be defined